source/include/CEntryPoints.h

```diff
@@ -21,6 +21,9 @@ typedef enum {
     kEggShellResult_UnexpectedObjectType = 2,
     kEggShellResult_InvalidResultPointer = 3,
     kEggShellResult_UnableToCreateReturnBuffer = 4,
+    kEggShellResult_InvalidTypeRef = 5,
+    kEggShellResult_MismatchedArrayRank = 6,
+    kEggSehllResult_UnableToParseData = 7,
```

Comment:
Typo here

Reply:
Thanks for catching this one. I'll fix it 👍